usr/local/bin/grant_access

```
@@ -78,5 +120,14 @@ def get_args():
     return parser.parse_args()
 
 
+def valid_email(s):
+    user, _, domain = s.rpartition('@', 1)
+    return EMAIL_USER.match(user) and EMAIL_DOMAIN.match(domain)
+
+
+def parse_emails(emails):
+    return list(filter(valid_email, re.split(DELIMITERS, emails)))
```

If we just filter out invalid emails, don't we risk to not notice typos, etc, in this list?

I wonder if it would be better to fail loudly and allow the user to fix the emails rather than ignore the errors and continue. Obviously, this is not wrong or right, it's just a design decision. Worth thinking about it.